src/docs/sdk/features.mdx

```diff
@@ -268,6 +268,12 @@ If Performance Monitoring is both supported by the SDK and enabled in the client
 - span status must match HTTP response status code ([see Span status to HTTP status code mapping](/sdk/event-payloads/span/))
 - when network error occurs, span status must be set to `internal_error`
 
+The `url` in breadcumbs and `$url` in span descriptions must be stripped of sensitive data like query parameters and/or username/password.
```

Comment:
I suspect that URL info could be in different places as well, See [this](https://github.com/getsentry/sentry/blob/fef9c695a1a7d3384fb3ce7ec6c264632e77061d/src/sentry/interfaces/spans.py#L34).